.evergreen/run-tests.sh

```diff
@@ -39,8 +39,15 @@ if [ "$SANITIZE" != "false" ]; then
     SANITIZE_STATEMENT="--sanitize ${SANITIZE}"
 fi
 
+# TODO SWIFT-1421: remove this, it is currently needed due to a bug in Xcode 13.0/13.1.
```

Comment:
I believe this has been fixed in newer Xcode versions as I haven't ever encountered it locally myself, but unfortunately we still need the build team to install newer Xcode on the macOS 11 hosts